test/src/test/java/hudson/model/ParametersTest.java

```diff
@@ -66,7 +66,7 @@ public void parameterTypes() throws Exception {
 
         HtmlElement element = (HtmlElement) ((HtmlElement) DomNodeUtil.selectSingleNode(form, "//div[input/@value='string']")).getParentNode();
         assertNotNull(element);
-        assertEquals("string description", ((HtmlElement) DomNodeUtil.selectSingleNode(element.getNextSibling().getNextSibling(), "div[@class='setting-description']")).getTextContent());
+        assertEquals("string description", element.getParentNode().querySelector(".jenkins-form-description").getTextContent());
```

Comment:
this test is so fragile to markup, every change breaks it 😢

Reply:
Should be able to make it more stable with https://github.com/jenkinsci/jenkins/pull/5926